pkgs/GitGuardian/ggshield/registry.yaml

```diff
@@ -0,0 +1,28 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/aquaproj/aqua/main/json-schema/registry.json
+packages:
+  - type: github_release
+    repo_owner: GitGuardian
+    repo_name: ggshield
+    description: Detect and validate 500+ types of hardcoded secrets with advanced checks. Use it as a pre-commit hook, GitHub Action, or CLI for proactive secret detection and security
+    version_filter: semver("> 1.30.2")
+    version_constraint: "false"
+    version_overrides:
+      - version_constraint: "true"
```

Comment:
If you want to drop the support of old versions, please use `no_asset` or `error_message`:

```yaml
    version_constraint: "false"
    version_overrides:
      - version_constraint: semver("<= 1.30.2")
        error_message: This version is too old. Please update this to 1.30.3 or later.
      - version_constraint: "true"
```

Reply:
Seems like we can support old versions too.
Why do you drop them?

https://github.com/GitGuardian/ggshield/releases/tag/v1.30.1

Reply:
They released macOS binaries with 1.31.0 so it was just to simplify the logic but I can rework to support non-macos binaries prior to 1.31.0